test/interpreter/Variables.test.js

```diff
@@ -100,11 +100,11 @@ describe("interpreter variables", () => {
             },
             {
                 lhs: "double#",
-                values: [str, int32, float, int64],
+                values: [str, int32, int64, ""],
```

Comment:
what are the empty strings for? will strings be casted to doubles?

Reply:
Here are described types that are not allowed for `Double`. So, I removed `float` cuz we made changes to cast a `float` to `Double`. So, we expect that the test should not fail if we get `float`  or `Double` value for a variable with `Double` type.

Reply:
I get the part of removing `float` from the list but what about the last empty string in the array? I guess we already check for string types in the beginning of the array so why is `""` needed? Also `""` is not a BrsType, it would be `new BrsString("")`

Reply:
My bad, removing it.